server/routes/pubDocument.tsx

```diff
@@ -104,6 +107,17 @@ const getEnrichedPubData = async ({
 		pubData.reviewHash = reviewHash;
 	}
 
+	if (!pubData.commentHash) {
```

Comment:
It's generally better to do this sort of thing as a migration so the codebase doesn't accumulate these ad-hoc backfills. If you keep adding them eventually you get to a point where it's hard to reason about the state of any Pub. "Just one" is okay but we should really strive to hold the line on this.